### DISCUSSION TITLE
Enrolling in GitHub Team: How is the "This organization belongs to" info used?

### DISCUSSION THREAD

**Discussion by ItaiBEdu** [community/community#65158] [GitHub Education] [answered]
I am a computer science high-school teacher. As such, I have enrolled to the GitHub education program (and have been approved). I was informed that my membership in the program makes me eligible for a free GitHub Team membership. I would like to enrol in GitHub Team.

Once I accessed [the GitHub Team homepage](https://github.com/team) and clicked the "Get started with Team" button, I was presented with a form (see the following screenshot) requesting me to enter a few pieces of information

![GitHubTeamSetUpOrg](https://github.com/community/community/assets/143313920/6669599f-56ba-4f29-848e-93b1f32a7e82)

One of the mandatory pieces of information I am required to provide is an indication of whether my organization belongs to my personal account or to a business or institution.

Why is this piece of information mandatory in order to enrol in GitHub Team? In what way is this information used?

I don't know which of the two alternatives (personal account/business or institution) to select. On the one hand, my organization, namely the high-school in which I teach, is an institution. On the other hand, as far as I know my high-school does not have an independent GitHub account. As far as I know I am the only staff member who has created an education GitHub account, and therefore, in a sense, my high-school "lives" in my personal account.

In order to select the appropriate alternative, it would help me to know the answers to the two questions posed above:

1. Why is this piece of information (i.e. whether my organization belongs to my personal account or to a business or institution) mandatory in order to enrol in GitHub Team?
2. In what way is the information described in the first question used by GitHub, and in particular by the process that enrols users to GitHub Team?

**Comment by arafajs** [comment, marked as answer]
The information about whether your organization belongs to your personal account or to a business or institution is mandatory in order to enroll in GitHub Team because GitHub needs to know how to manage your organization's access to GitHub's resources.

If you select "personal account", GitHub will create a new organization for your high school and give you the role of organization owner. This means that you will have full control over the organization's settings and permissions.

If you select "business or institution", GitHub will assume that your high school already has an organization on GitHub. In this case, you will need to provide the organization's name and contact information. GitHub will then verify that you are authorized to create a team for the organization.

In your case, since your high school does not have an independent GitHub account, you should select "personal account". This will ensure that you have full control over the organization's settings and permissions.

**Reply by chacken** [reply]
They should put this explanation on that page, because they give you zero context otherwise.

**Comment by Sharma-Mayank15** [comment]
GitHub uses this info to determine how to structure access, ownership, and administrative permissions for your organization:

If you select "Personal Account":
GitHub understands that you are the sole administrator of this organization. It will create a new GitHub Organization under your account and assign you as the owner, giving you full control over repository management, member roles, billing (if any), and permissions. This option is ideal for solo educators managing student projects independently.

If you select "Business or Institution":
GitHub assumes the organization is already formally established (e.g., your high school has its own GitHub organization/account). In that case, GitHub may require additional contact info and verification to ensure you're authorized to represent and manage the team within that existing structure.

**Reply by olivertwist-code** [reply]
yeah it is
